app/views/confirmation.scala.html

```diff
@@ -24,7 +24,7 @@
     bodyClasses = None) {
 
     <div class="govuk-box-highlight">
-        @components.heading("confirmation.heading", "heading-xlarge")
```

Comment:
`heading-xlarge` is the default value in `heading.scala` - so need to specify it here